pims/image_sequence.py

```diff
@@ -16,6 +19,7 @@
 
 # skimage.io.plugin_order() gives a nice hierarchy of implementations of imread.
 # If skimage is not available, go down our own hard-coded hierarchy.
+from PIL import Image
```

Comment:
I tested it with Pillow only, but I think `Image.open` has the same syntax in both versions.